feedback/lib/src/feedback_bottom_sheet.dart

```diff
@@ -2,34 +2,140 @@
 
 import 'package:feedback/src/better_feedback.dart';
 import 'package:feedback/src/theme/feedback_theme.dart';
+
 import 'package:flutter/material.dart';
 
 /// Shows the text input in which the user can describe his feedback.
 class FeedbackBottomSheet extends StatelessWidget {
   const FeedbackBottomSheet({
     Key? key,
-    required this.feedbackBuilder,
+    required this.feedbackSheetBuilder,
     required this.onSubmit,
+    required this.sheetProgress,
   }) : super(key: key);
 
-  final FeedbackBuilder feedbackBuilder;
+  final FeedbackSheetBuilder feedbackSheetBuilder;
   final OnSubmit onSubmit;
+  final ValueNotifier<double> sheetProgress;
 
   @override
   Widget build(BuildContext context) {
-    // We need to supply a navigator so that the contents of the bottom sheet
-    // have access to an overlay (overlays are used by many material widgets
-    // such as `TextField` and `DropDownButton`.
-    // Typically, the navigator would be provided by a `MaterialApp`, but
-    // `BetterFeedback` is used above `MaterialApp` in the widget tree so that
-    // the nested navigation in navigate mode works properly.
-    return Navigator(
-      onGenerateRoute: (_) => MaterialPageRoute<void>(
-        builder: (context) => Material(
+    // We need to supply an navigator so that the contents of the bottom sheet
+    // have access to it. Overlays are used by many material widgets
+    // such as `TextField` and `DropDownButton`. Typically, this would be
+    // provided by `MaterialApp`, but `BetterFeedback` is above `MaterialApp` in
+    // the widget tree.
```

Comment:
This comment seems now out of date, right? Don't we need a Navigator anymore?

Reply:
FIxed.